### ISSUE TITLE
Allow headings to be aligned

### ISSUE THREAD

**Issue by wsidl** [asciidoctor/asciidoctor-pdf#399]
Would it be possible to add an option for headings? You can align the base style and other block types, but headings don't have the option.

An option would be similar to this:
````yaml
heading:
    h2:
        font_size: 16
        align: center
````

**Comment by mojavelinux** [comment]
Definitely. Do we need to support alignment per level or just all headings with the same alignment? Also, what about the alignment for the Table of Contents heading? Should that be configured separately or part of the setting for headings? (currently it's coupled to the level-2 heading). Maybe it's time to split that out into `toc -> title` (like `sidebar -> title`).

**Comment by wsidl** [comment]
Would be preferred if it was per heading level. I have some documents where levels 2-4 are center aligned and level 5 is left aligned.

I would be fine if the macro headers (TOC, table of figures, etc) were kept tied to level 2. I'm sure someone would disagree though.

**Comment by mojavelinux** [comment]
Thanks for that input. That's exactly what I needed to know.

**Comment by mojavelinux** [comment]
This is a duplicate of #343 and was resolved in 1.5.0.alpha.12.
